charts/egeria-platform/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Egeria Settings
+egeria:
+  # Global Egeria Settings
+  logging: error
+  version: "3.5"
+  debug: true
+
+  # persistence
+  persistence: true
+  storageClass: ibmc-vpc-block-10iops-tier
```

Comment:
this is deployment specific. Suggest we just go with default storage class & allow chart user to override
Also 8Gi may be optimistic in a local/test environment. Probably would err on a smaller default & make configurable (not sure if lazy allocation)

Reply:
Fixed in 770da2ec22a0